crates/db_views/src/post_view.rs

```diff
@@ -178,7 +173,7 @@ impl<'a> PostQueryBuilder<'a> {
   pub fn create(conn: &'a PgConnection) -> Self {
     PostQueryBuilder {
       conn,
-      listing_type: None,
+      listing_type: Some(ListingType::All),
```

Comment:
I dont think there is any reason for this to be an option now.

Reply:
Check out the fields on all of these, they're all options.

Reply:
Sure but thats not a good reason to make this one an option too. Option should only be used when its required, just like other features.

Reply:
Looks like i will rewrite this anyway in https://github.com/LemmyNet/lemmy/pull/2379, so this doesnt matter.